bundles/org.openhab.binding.homewizard/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -98,6 +131,68 @@
 
 	</thing-type>
 
+	<thing-type id="energy_socket">
+		<label>HomeWizard Energysocket</label>
+		<description>This thing provides the measurement data that is available through the http interface of a HomeWizard
+			Energysocket.</description>
+
+		<channels>
+			<channel id="total_energy_import_t1" typeId="total_energy_import_t1"/>
+			<channel id="total_energy_export_t1" typeId="total_energy_export_t1"/>
+			<channel id="active_power" typeId="active_power"/>
+
+			<channel id="power_switch" typeId="power_switch"/>
+			<channel id="power_lock" typeId="power_lock"/>
+			<channel id="ring_brightness" typeId="ring_brightness"/>
+
+		</channels>
+
+		<config-description>
+			<parameter name="ipAddress" type="text" required="true">
+				<label>Network Address</label>
+				<description>The IP or host name of the Energysocket.</description>
+				<context>network-address</context>
+			</parameter>
+			<parameter name="refreshDelay" type="integer" min="1" unit="s">
+				<label>Refresh Interval</label>
+				<description>The refresh interval in seconds for polling the Energysocket.</description>
+				<default>5</default>
+			</parameter>
+		</config-description>
+
+	</thing-type>
+
+	<thing-type id="watermeter">
```

Comment:
This thing type is already defined in line 109?

Reply:
Merge conflicts leftover. Fixed.